packaging/usr/share/bash-completion/completions/pikaur

```diff
@@ -50,6 +50,46 @@ _pacman_pkg() {
   )"
 }
 
+_pikaur_get_aur_packages() {
+  local cacheFile=$1
+  local aurPackagesURL='https://aur.archlinux.org/packages.gz'
+  local tmpFile
+  tmpFile="$(mktemp)"
+  curl -f -s "${aurPackagesURL}" -o "${tmpFile}"
+  local returnStatus=$?
+  if [ "${returnStatus}" -eq 0 ]; then
+    mv "${tmpFile}" "${cacheFile}" 2>/dev/null
+  else
+    rm "${tmpFile}"
+  fi
+}
+
+_pikaur_cached_search() {
+  local repoPackageNames
+  repoPackageNames="$(\pacman -Ssq)" # pacman is faster then pikaur -Ssqo
```

Comment:
are you sure you need a backslash before `pacman`?

Reply:
I'm not sure.
For some reason there are other calls with backslash.
https://github.com/actionless/pikaur/blob/0f3786d33884bc79082fd9b1d83072f87cf39560/packaging/usr/share/bash-completion/completions/pikaur#L31
https://github.com/actionless/pikaur/blob/0f3786d33884bc79082fd9b1d83072f87cf39560/packaging/usr/share/bash-completion/completions/pikaur#L46

Reply:
must be some bash occultism

Reply:
> Use a “\” (backslash) before the command to run it without the alias.